show_res: false
camera:
  type: "realsense" # realsense or zed
  intrin:
    fx: 643.898
    fy: 643.216
    cx: 649.038
    cy: 357.21
    distortion_coeffs: [-0.0553056,0.065975,-0.000994232,2.98548e-05,-0.0216579]
    distortion_model: 2 # 0: none, 1: opencv format, 2: for rs d455 only
  extrin:
    - [ 0.05659255, 0.03014561, 0.99794215, 0.04217854]
    - [-0.99839673, 0.00283376, 0.05653273, 0.01405556]
    - [-0.00112372,-0.9995415 , 0.03025765,-0.01538294]
    - [ 0.        , 0.        , 0.        , 1.        ]
  pitch_compensation: 0.0 # in degree
  yaw_compensation: 0.0
  z_compensation: 0.0
detection_model:
  model_path: "./src/vision/model/best_orin.engine"
  confidence_threshold: 0.2
use_depth: true
ball_pose_estimator:
  use_depth: false
  radius: 1.109
  down_sample_leaf_size: 0.01
  cluster_distance_threshold: 0.01
  fitting_distance_threshold: 0.01
human_like_pose_estimator:
  use_depth: false
  down_sample_leaf_size: 0.01
  fitting_distance_threshold: 0.01
  statistic_outlier_multiplier: 0.01
  
